snap/snapcraft.yaml

```
@@ -340,9 +340,14 @@ apps:
 parts:
   version:
     plugin: nil
-    source: .
+    # as with curl plugin, the source dir is unrelated to this part and is used
+    # since it changes rarely and therefore will not trigger a new pull
+    source: snap/local/build-helpers
     override-pull: |
-      snapcraftctl set-version $(cat $SNAPCRAFT_PROJECT_DIR/VERSION)-$(date +%Y%m%d)+$(git rev-parse --short HEAD)
+      cd $SNAPCRAFT_PROJECT_DIR
+      PROJECT_VERSION=$(cat VERSION)
+      GIT_REVISION=$(git rev-parse --short HEAD)
+      snapcraftctl set-version ${PROJECT_VERSION}-$(date +%Y%m%d)+${GIT_REVISION}}
```

same as the master branch, need to drop the last `}`:

```suggestion
      snapcraftctl set-version ${PROJECT_VERSION}-$(date +%Y%m%d)+${GIT_REVISION}
```